drivers/adc/Kconfig.mcux

```diff
@@ -11,3 +11,37 @@ config ADC_MCUX_ADC16
 	depends on HAS_MCUX_ADC16
 	help
 	  Enable the MCUX ADC16 driver.
+
+if ADC_MCUX_ADC16
+
+choice
+	prompt "Clock Divide Selection"
```

Comment:
The clock divider and voltage reference should probably come from device tree instead.

Reply:
That was my first thought. Kconfig was then more convenient to try out another configuration. Also, the settings could differ from sample to sample.